lib/Cluster.js

```diff
@@ -991,6 +1058,42 @@ class Cluster extends EventEmitter {
     }
   }
 
+  /**
+   * Given an array of attribute ids it checks if all of the attributes are manufacturer
+   * specific attributes. If that is the case it will return the manufacturerId so that the
+   * command can set the manufacturerSpecific flag.
+   * @param {number[]} attributeIds
+   * @returns {null|number}
+   * @private
+   */
+  _checkForManufacturerSpecificAttributes(attributeIds) {
+    // Convert to set
+    const attrIdsSet = new Set(attributeIds);
+
+    // Filter attributeIds for manufacturer specific attributes
+    const manufacturerIds = Object.entries(this.constructor.attributes)
+      .reduce((acc, [key, value]) => {
+        if (attrIdsSet.has(value.id) && typeof value.manufacturerId === 'number') acc.push(value.manufacturerId);
+        return acc;
+      }, []);
+
+    // Do not allow different manufacturer ids in one command
+    if (new Set(manufacturerIds).size > 1) {
+      throw new Error('Error: detected multiple manufacturer ids, can only read from one at a'
+        + ' time');
```

Comment:
did eslint force this? don't like strings "cut" like this :/

Reply:
Hm no, it seems to be my IDE doing this :\ Anyway, fixed.